test/e2e/e2e-data/src/main/java/org/apache/skywalking/e2e/metrics/MetricsQuery.java

```diff
@@ -69,13 +69,31 @@
         SERVICE_INSTANCE_SLA
     };
 
+    public static String INSTANCE_JVM_CPU = "instance_jvm_cpu";
+    public static String INSTANCE_JVM_MEMORY_HEAP = "instance_jvm_memory_heap";
+    public static String INSTANCE_JVM_MEMORY_HEAP_MAX = "instance_jvm_memory_heap_max";
+    public static String INSTANCE_JVM_MEMORY_NOHEAP = "instance_jvm_memory_noheap";
+    public static String INSTANCE_JVM_MEMORY_NOHEAP_MAX = "instance_jvm_memory_noheap_max";
+    public static String INSTANCE_JVM_YOUNG_GC_TIME = "instance_jvm_young_gc_time";
+    public static String INSTANCE_JVM_OLD_GC_TIME = "instance_jvm_old_gc_time";
+    public static String INSTANCE_JVM_YOUNG_GC_COUNT = "instance_jvm_young_gc_count";
+    public static String INSTANCE_JVM_OLD_GC_COUNT = "instance_jvm_old_gc_count";
```

Comment:
I think we can't check all of the metrics, because some of them have chance to be none in the e2e test env. Such as CPU and GC.

Reply:
Updated,  thanks